php/tests/encode_decode_test.php

```diff
@@ -31,6 +31,7 @@ public function testDecodeJsonSimple()
     {
         $m = new TestMessage();
         $m->mergeFromJsonString("{\"optionalInt32\":1}");
+        $this->assertTrue(true);
```

Comment:
Why is this necessary? What about asserting that `$m->getOptionalInt32()` equals `1`?

Reply:
Change this to assert equals 1.
For the other assertTrue, higher version of phpunit want to have at least one assertion. Otherwise, it will throw warnings.